internal/blocktx/processor.go

```diff
@@ -386,6 +407,7 @@ func (p *Processor) publishMinedTxs(txHashes []*chainhash.Hash) error {
 			BlockHash:       minedTx.BlockHash,
 			BlockHeight:     minedTx.BlockHeight,
 			MerklePath:      minedTx.MerklePath,
+			BlockStatus:     minedTx.BlockStatus,
 		}
 		err = p.mqClient.PublishMarshal(MinedTxsTopic, txBlock)
```

Comment:
That is unrelated to the PR, but here we only return the err if it happens in the last iteration.
I think it should be at each iteration

```go
	for _, minedTx := range minedTxs {
		txBlock := &blocktx_api.TransactionBlock{
			TransactionHash: minedTx.TxHash,
			BlockHash:       minedTx.BlockHash,
			BlockHeight:     minedTx.BlockHeight,
			MerklePath:      minedTx.MerklePath,
			BlockStatus:     minedTx.BlockStatus,
		}
		err = p.mqClient.PublishMarshal(MinedTxsTopic, txBlock)
	        if err != nil {
		    return fmt.Errorf("failed to publish mined transactions: %v", err)
	        }
       }
```

Reply:
I think we don't return here on purpose:
1. If there's a problem sending one transaction, we still probably want to send the rest of them
2. If we return an error from this method, we would considered the block unprocessed, when in reality it's processed, but there's only a problem with transaction.

I think it's okay the way it is, but let me know if I'm not missing something.